c/pthread-C-DAC/LICENSE.txt

```diff
@@ -0,0 +1 @@
+email: hpcfte AT cdac.in
```

Comment:
The fact that they told you that you can use the benchmarks means that you have a license (because such a permission is a license). It is just not one of those formalized well-known licences.

For this repository, it is important to document what requirements are attached to the benchmarks and what permissions do we have. So you would need to state this precisely in the license file. This would be much easier if they just told you that you can use it under some well-known license, but in absence of this please provide as much information as possible. Then we can check whether these permissions are enough to incorporate the benchmarks in this repository (remember that everything that is not explicitly granted is not allowed).